templates/registration/donorRegistration/donorBilling.jinja

```diff
@@ -6,45 +6,57 @@
 
 {% block body %}
 <div class="formContainer">
+<form action="/newsFeed">
     <div class="form-group">
         <br>
         <label for="">Billing Information<br>(optional)</label>
-        <button class="btn btn-purple rightBox" style="width: 50px">Skip</button>
+        <a href="/newsFeed"><button type="button" class="btn btn-purple rightBox" style="width: 50px">Skip</button></a>
```

Comment:
Not sure if there's a reason to explicitly give it width. If not, remove that entirely. If there is, it's width should be 53px.